curation_concerns-models/app/models/concerns/curation_concerns/generic_work_behavior.rb

```diff
@@ -24,7 +24,13 @@ def indexer
   end
 
   def to_s
```

Comment:
@njaffer I think it might be easier to copy the to_s method exactly from [serializers.rb](https://github.com/projecthydra-labs/curation_concerns/blob/e1bb116e14ed5417a6950fb8ff1decbea0f57714/curation_concerns-models/app/models/concerns/curation_concerns/serializers.rb)